bindings/python/src/metadata.rs

```diff
@@ -90,6 +90,18 @@ impl Metadata {
     pub fn mode(&self) -> EntryMode {
         EntryMode(self.0.mode())
     }
+
+    pub fn __repr__(&self) -> String {
+        format!(
+            "Metadata(content_disposition={}, content_length={}, content_md5={}, content_type={}, etag={}, mode={})",
```

Comment:
`content_disposition` might be empty most of the time, so I don't think it's worth printing in `repr`. How about this instead:

```
"Metadata(mode={}, content_length={}, content_type={}, etag={})"
```

We arrange the metadata by importance and ignore less frequently used metadata.

Reply:
cool will drop this, you are right.

and FYI last_modified maybe needed here too?

Reply:
Makes sense to me.

Reply:
Cool will update this patch after last_modified Done

Reply:
Done and lerned that, thanks